### PR TITLE
Link the fund/lab on a submission detail viewto the public fund/lab page

### LINKED ISSUE #610: On the submission page, where it says the request or fund the submission came from, make that a link to the public facing page for the fund or request (not the application itself)

### PR DESCRIPTION
**Pull request by frjo**
Fixes #610

### CODE REVIEW

File: opentech/apply/funds/templates/funds/applicationsubmission_detail.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
         <h1 class="beta heading heading--no-margin heading--bold">{{ object.title }}</h1>
         <h5 class="heading heading--meta">
             <span>{{ object.stage }}</span>
-            <span>{{ object.page }}</span>
+            <span><a class="link--transparent link--underlined" href="{% pageurl public_page %}" >{{ object.page }}</a></span>
```
Comment (todd-dembrey) [inline opentech/apply/funds/templates/funds/applicationsubmission_detail.html]: can you not just `{% pageurl object.page %}`?

Comment (frjo) [reply opentech/apply/funds/templates/funds/applicationsubmission_detail.html]: That links to the form and they want the public page for the form.

File: opentech/apply/funds/views.py [hunk 2]
```diff
@@ -153,8 +155,15 @@ def get_context_data(self, **kwargs):
         if self.object.next:
             other_submissions = other_submissions.exclude(id=self.object.next.id)
 
+        try:
+            public_page = BaseApplicationPage.objects.get(application_type_id=self.object.page.id)
```
Comment (todd-dembrey) [inline opentech/apply/funds/views.py]: You should be able to use `self.object.get_from_parent('detail')()` which exists for both labs and applications

https://github.com/OpenTechFund/opentech.fund/blob/9fd79c7135d8e3fb003c0abe8418f1cac68b0438/opentech/apply/funds/models/applications.py#L60-L62

Comment (frjo) [reply opentech/apply/funds/views.py]: Works perfectly, thanks.

**Pull request merged**